fs/exists.ts

```diff
@@ -27,39 +59,89 @@
  * }
  * ```
  * @see https://en.wikipedia.org/wiki/Time-of-check_to_time-of-use
- * @deprecated (will be removed after 0.157.0) Checking the state of a file before using it causes a race condition. Perform the actual operation directly instead.
  */
-export async function exists(filePath: string | URL): Promise<boolean> {
+export async function exists(
+  path: string | URL,
+  options?: ExistsOptions,
+): Promise<boolean> {
   try {
-    await Deno.lstat(filePath);
+    const stat: Deno.FileInfo = await Deno.stat(path);
```

Comment:
type annotation seems unnecessary as it's inferred from the assigned value

```suggestion
    const stat = await Deno.stat(path);
```

Can you also remove this kind of type annotations for local variables?